scripts/add_nonce.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+# This must be called from repo root as current directory
+
+NONCE=$(date +%s)
+
+for packagedir in ./packages/*; do
+  if [[ -d "$packagedir" ]]; then
+    echo "$packagedir"
+    mkdir -p "$packagedir/nonces"
+    echo "Directory used to trigger lerna package updates for all packages" \
```

Comment:
I guess we can remove this now.

Reply:
The command can remain included to cover new packages or README updates as well